### ISSUE TITLE
Overflow on Details tab

### ISSUE THREAD

**Issue by talbert83** [codecentric/spring-boot-admin#562]
Hi, I'm evaluating SBA 1.5.4 for use in our system and noticed another issue. We have some custom data exposed through the /info endpoint and it's being rendered in the "Details" tab in the "Application" section. Unfortunately some entires contain quite a lot of text. For instance I'm storing a list with info about the last n errors in the application. This is not rendered very nicely as the text overflows (at least on my screen). 

Here's the json from the /info endpoint:
```
- errorMessages: [
    {
    method: "GET",
    uri: "/api/search",
    status: 500,
    response: "[ { "id" : "----", "message" : "Unknown error", "qualifier" : "error" } ]",
    occurrence: "2017-09-27T16:18:43.011"
    }
]
```
And here's a screenshot of what it looks like in the UI:
![image](https://user-images.githubusercontent.com/32326871/30916564-64dfcad0-a3a2-11e7-8683-e8a3140a207a.png)
The div has width: 400px which exceeds the width of the enclosing section.

The fix would be to restrict the width of the inner divs to that of the section, but I'm not good at frontend stuff to provide a fix myself. Until then I'll try to output the value of "response" as json instead of string, see what happens - maybe it solves my problem.

**Comment by joshiste** [comment]
Ok I can see the issue, but I don't think this fix will make it into a 1.5.x version as the UI will get an overhaul for 2.x.

**Comment by ismarslomic** [comment]
Just want to add that overflow is an issue for the main page as well. For instance if the application URL is very long, it is not rendered nicely. Let me know if you need a screen shot and you cant reproduce this.
